symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -37,6 +37,7 @@ public function registerBundles()
     protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader)
     {
         $container->setParameter('container.autowiring.strict_mode', true);
+        $container->setParameter('container.dumper.inline_class_loader', true);
```

Comment:
Can't it be set to `on` by default in 4.0?

Reply:
it is in the flex recipe, would be a hard BC break otherwise

Reply:
We should still work on making this on by default in the future (we could trigger a deprecation notice in 3.4 if the parameter is not set explicitly to either `true` or `false`, saying that the default behavior in 4.0 would be to enable this).

`container.autowiring.strict_mode` does not suffer from this, because it is just about disabling a BC layer (so it is gone entirely in 4.0)